pkg/sdn/plugin/api/types.go

```diff
@@ -1,17 +1,22 @@
 package api
 
 import (
-	knetwork "k8s.io/kubernetes/pkg/kubelet/network"
 	pconfig "k8s.io/kubernetes/pkg/proxy/config"
-)
-
-type OsdnNodePlugin interface {
-	knetwork.NetworkPlugin
 
-	Start() error
-}
+	cnitypes "github.com/containernetworking/cni/pkg/types"
+)
 
 type FilteringEndpointsConfigHandler interface {
 	pconfig.EndpointsConfigHandler
 	Start(baseHandler pconfig.EndpointsConfigHandler) error
 }
+
+// Standard CNI network configuration block written to /etc/cni/net.d
+// which is picked up by the Kubernetes CNI network plugin and used to
+// run our openshift-sdn CNI plugin
+type CNINetConfig struct {
```

Comment:
In the past, pkg/sdn/api/ was only for API that was exported to the rest of origin, whereas this is just shared between the core plugin code and the CNI plugin...